### PR TITLE
Issue 269, dutch text

### PR DESCRIPTION
**Pull request by elma13**
The Dutch text has been updated, after some review of colleagues. English translation will be next issue.

### CODE REVIEW

File: lang/translation.json [hunk 1]
```diff
@@ -376,51 +410,51 @@
...
+        "nl": "Hier zie je de gemiddelde concentratie op elk uur van de dag. De staafjes geven de spreidng weer, meer informatie is te vinden onder het 'Informatie over de tool'-tabblad."
       },
       {"tag" : "word_indu_timeseries",
         "en": "Individual timeseries",
         "nl": "Individuele tijdreeks"
       },
       {"tag" : "expl_indu_timeplot",
         "en": "Here you can view the timeseries of the individual sensor.",
-        "nl": "Hier kunt u de tijdreeks van de individuele sensor bekijken."
+        "nl": "Op deze pagina kun je de metingen van losse sensoren bekijken uit de selectie die je net gemaakt hebt. Daarnaast kun je de maximale waarde instellen, meetwaardes boven deze waarde worden dan niet meer getoond. Vaak zijn de hoge metingen gemeten met een vieze of stukke sensor. Hierdoor niet realistisch voor de buitenlucht."
```
Comment (jspijker) [inline lang/translation.json:425]: 'stukke sensor', is that correct Dutch? Last sentence is missing something

File: lang/translation.json [hunk 2]
```diff
@@ -80,9 +71,18 @@
...
 	      "nl": "Links"
 	  },
+	  {
+	    "tag" : "word_maximum_value",
+	    "en" : "Filter on the maximal value",
+	    "nl" : "Filteren op maximale waarde"},
+	    {
+	      "tag" : "tool_maximum_value_1_expl",
+	      "en" : "",
+	      "nl": "Elke meetwaarde die de sensor doorgeeft kan informatie geven over de luchtkwaliteit van die plek. Echter, wanneer de sensor vies of beschadigd is, is de meetwaarde niet representatief voor de luchtkwaliteit. Vaak zijn dat dan hoge waardes van bijvoorbeeld 300. Voor de analyse wil je graag de luchtkwaliteit bekijken, zonder dat deze verstoord wordt door deze hoge waardes. Daarvoor kun je een filter zetten die de meetwaardes boven de maximale waarde niet tonen. Standaard staat de maximale waarde op het 10% percentiel van de sensormetingen. Je kunt ook zelf het filter instellen, zo kun je ook de hoge waardes van een sensor bekijken. Zet de maximale dan bijvoorbeeld op 1000 en bekijk de sensoren één voor één."
```
Comment (jspijker) [inline lang/translation.json:81]: '10% percentiel' moet waarschijnlijk 90-precentiel zijn (en/of ' op de waarde waaronder 90% van de metingen vallen')

**Comment by jspijker** [comment]
When changing language in the 'overview'/'overzicht' tab, weird things happen. In my case when switching from Dutch to English, I loose the 'step' 1 to 3 tabs. Possibly the English translation of the 'stap' labels is still missing?

**Review by jspijker — CHANGES_REQUESTED** [review]

**Comment by elma13** [comment]
> When changing language in the 'overview'/'overzicht' tab, weird things happen. In my case when switching from Dutch to English, I loose the 'step' 1 to 3 tabs. Possibly the English translation of the 'stap' labels is still missing?

This will probably be fixed after the english translation, #273

File: lang/translation.json (hunk 1 above)
Comment (elma13) [reply lang/translation.json:425]: Oke, I changed it to "kapotte"

File: lang/translation.json (hunk 2 above)
Comment (elma13) [reply lang/translation.json:81]: Thanks, I changed it to 90%

File: lang/translation.json (hunk 1 above)
Comment (elma13) [reply lang/translation.json]: Senstence rewritten to have a better sentence

**Review by jspijker — APPROVED** [review]

**Pull request merged**